src/main/kotlin/com/terraformation/backend/customer/db/ProjectStore.kt

```diff
@@ -84,6 +84,7 @@ class ProjectStore(
           .set(PROJECTS.MODIFIED_BY, currentUser().userId)
           .set(PROJECTS.MODIFIED_TIME, clock.instant())
           .set(PROJECTS.NAME, updated.name)
+          .where(PROJECTS.ID.eq(updated.id))
```

Comment:
This should use `projectId`; otherwise the update function can return a model with a different project ID and we'll update the wrong one.